src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -194,6 +194,17 @@ describe('InputDropdownComponent', () => {
         expect(openChangeSpy).not.toHaveBeenCalled();
     });
 
+    it('should not focus the input field when dropdownOpen is programmatically set to true', async () => {
+        component.dropdownOpen = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        const input = document.querySelector<HTMLInputElement>('input.form-control');
+
+        expect(document.activeElement).not.toBe(input);
```

Comment:
I reverted your fix below and ran this test and it still passed. Could you look into another way to verify this, and check that it is effective?

Reply:
Ignore this, I made a mistake.